<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchActionDialog</class>
 <widget class="QDialog" name="BatchActionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Action</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Active Batch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxBatches">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRunBatch">
       <property name="text">
        <string>Run</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/batch/res/play.png</normaloff>:/batch/res/play.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <property name="rowCount">
      <number>100</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>180</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>75</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string notr="true">Who</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Do What</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">Parameter</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonNewBatch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>New</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/batch/res/new.png</normaloff>:/batch/res/new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeleteBatch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/batch/res/delete.png</normaloff>:/batch/res/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRenameBatch">
       <property name="text">
        <string>Rename</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/batch/res/rename.png</normaloff>:/batch/res/rename.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCloneBatch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Clone</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/batch/res/copy.png</normaloff>:/batch/res/copy.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonModifyBatch">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Modify</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/batch/res/edit.png</normaloff>:/batch/res/edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSaveBatch">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/batch/res/save.png</normaloff>:/batch/res/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BatchActionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BatchActionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNewBatch</sender>
   <signal>clicked()</signal>
   <receiver>BatchActionDialog</receiver>
   <slot>onNewBatch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDeleteBatch</sender>
   <signal>clicked()</signal>
   <receiver>BatchActionDialog</receiver>
   <slot>onDeleteBatch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRenameBatch</sender>
   <signal>clicked()</signal>
   <receiver>BatchActionDialog</receiver>
   <slot>onRenameBatch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCloneBatch</sender>
   <signal>clicked()</signal>
   <receiver>BatchActionDialog</receiver>
   <slot>onCloneBatch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonModifyBatch</sender>
   <signal>clicked()</signal>
   <receiver>BatchActionDialog</receiver>
   <slot>onModifyBatch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSaveBatch</sender>
   <signal>clicked()</signal>
   <receiver>BatchActionDialog</receiver>
   <slot>onSaveBatch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>567</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRunBatch</sender>
   <signal>clicked()</signal>
   <receiver>BatchActionDialog</receiver>
   <slot>onRunBatch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>744</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onNewBatch()</slot>
  <slot>onDeleteBatch()</slot>
  <slot>onRenameBatch()</slot>
  <slot>onCloneBatch()</slot>
  <slot>onModifyBatch()</slot>
  <slot>onSaveBatch()</slot>
  <slot>onRunBatch()</slot>
 </slots>
</ui>
